reoptjl/models.py

```diff
@@ -437,32 +444,47 @@ class SiteOutputs(BaseModel, models.Model):
         primary_key=True
     )
 
-    annual_renewable_electricity_kwh = models.FloatField(
+    annual_onsite_renewable_electricity_kwh = models.FloatField(
         null=True, blank=True,
         help_text=(
                   "Electricity consumption (incl. electric heating/cooling loads) that is derived from on-site renewable resource generation."
                   "Calculated as total annual RE electric generation, minus storage losses and curtailment, with the user selecting whether exported renewable generation is included). "
                   )
     )
-    renewable_electricity_fraction = models.FloatField(
```

Comment:
did you want to remove that from the help text before merging?

Reply:
yes, done in [ecd9e7d](https://github.com/NREL/REopt_API/pull/628/commits/ecd9e7d5467a9fc5d50f1cd050b7d953887de608)